unity-renderer/Assets/DCLServices/WearablesCatalogService/LambdasWearablesCatalogService.cs

```diff
@@ -347,22 +347,26 @@ public async UniTask<IReadOnlyList<WearableItem>> RequestWearableCollection(IEnu
         }
 
         public async UniTask<IReadOnlyList<WearableItem>> RequestWearableCollectionInBuilder(IEnumerable<string> collectionIds,
-            CancellationToken cancellationToken, List<WearableItem> collectionBuffer = null)
+            CancellationToken cancellationToken, List<WearableItem> collectionBuffer = null, string nameFilter = null)
         {
             string domain = GetBuilderDomainUrl();
             var wearables = collectionBuffer ?? new List<WearableItem>();
 
             var queryParams = new[]
             {
                 ("page", "1"),
-                ("limit", "5000"),
+                // TODO: we should properly calculate pagination from the ui instead of requesting all at once
+                ("limit", "15000"),
```

Comment:
the request with this new limit is getting timeout from the backend, wdyt about implementing something like this:

```
int limit = 1000;
int total = 0;
int page = 1;

while (true)
{
    var queryParams = new[]
    {
        ("page", page),
        ("limit", limit),
    };

    ....
    (WearableCollectionResponseFromBuilder response, bool success) = await lambdasService.GetFromSpecificUrl<WearableCollectionResponseFromBuilder>(
    ... 
    total += response.data.results.Count;
    if (total >= response.data.total || page >= response.data.pages)
    {
        break;
    }

    page++;
}
```

Reply:
Reimplemented pagination properly